divinediscipline/send-it
# #161871329 Display profile page dynamically and make pages responsive across all devices

**What does this PR do?**
Display profile page dynamically

**Description of task to be completed**
- Display a users profile information dynamically
- make front end pages responsive across all devices

**How should this be manually tested?**
Clone the project, run 'npm install' to install all dependencies, run npm start:dev to start the server. navigate to localhost:5000 and login. check the users profile and test out application on a mobile device.

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
[Delivers #161871329]
